app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    stores = {}
+
+    def inner(*args: Any) -> Any:
+        if args in stores:
+            print("Getting from cache")
+        else:
```

Comment:
without ```else``` , please

Reply:
I don't understand how to write without else